docs/advanced-topics/reconnecting-mid-game.md

```diff
@@ -0,0 +1,28 @@
+---
+id: reconnecting-mid-game
+title: Reconnecting Mid-Game
+---
+
+In a multiplayer game, it is possible that a client gets disconnected from the server, for example if they have network issues, or if their application or device crashes. When it happens, you might want to give the ability to your players to reconnect to the game.
+
+To allow clients to reconnect in the middle of an active game, there are some considerations to have in mind.
+- If clients have a specific state and some in-game data associated with them and you want them to reconnect with that same state and that same data, you should implement a mechanism to keep that state in memory and associate it to that client when they reconnect (see [Session Management](./session-management.md))
```

Comment:
"In order to preserve a player's in-game state when they are disconnected, you will need to implement a mechanism to keep the player's in-game state in memory and restore that state upon the client reconnecting (see..."